apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -363,6 +374,19 @@ const DanceAiModal: React.FunctionComponent = () => {
     return currentProgress;
   };
 
+  // Handle moments during generation when we should play a sound.
+  useEffect(() => {
+    if (mode === Mode.GENERATING) {
+      if (generatingProgress.subStep === 2) {
+        if (generatingProgress.step < BAD_GENERATED_RESULTS_COUNT) {
+          playSound('ai-generate-no', {volume: 0.25});
+        } else {
```

Comment:
I'm not gonna block on this, but could it be 3 values `&&`ed together? I think that'd be easier to read than the if/else ladder, especially since we're not hopping out on only some conditions being true.

Reply:
Good point.  I think this is resolved in my next UI PR which builds on this one.

Reply:
I think we only want the sound to play when the first two conditions are true, but we could at least group those if it's clearer?